challenges/identification/I2_benchmark/I2_specification.md

```diff
@@ -0,0 +1,83 @@
+# I2 benchmark specification
+
+## Synopsis
+
+Benchmark to test sensitivity and False Discovery Rate (FDR) of RNAseq-based poly(A) site identification compared with ground truth (poly(A)-sites identified from orthogonal 3'end seq data).
+
+Input data:
+
+1. Poly(A) sites identified based on RNAseq data using the benchmarked tool
+2. Poly(A) sites identified based on orthogonal 3'end seq dataset
+
+Based on the input data the following metrics are computed:
+
+1. Sensitivity = (TP/(TP+FN))
+2. FDR = (FP/(TP+FP))
+
+TP - true positives - PAS identified by the tool and present in the orthogonal dataset  
+FP - false positives - PAS identified by the tool and not present in the orthogonal dataset  
+FN - false negatives - PAS not identified by the tool but present in the orthogonal dataset
+
+The metrics should be computed for the following distance threshold between PAS identified by the tool in RNAseq dataset and PAS identified in orthogonal 3'end seq dataset:
+
+- 50 nt
```

Comment:
I think 50 nt is good.

Reply:
So we can generate a curve based on varied distances between the algorithm predicted PAS and 3'end seq dataset PAS (Like in supplementary figure 1 a in this recent paper [https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-021-00864-5/MediaObjects/41588_2021_864_MOESM1_ESM.pdf](url)) would doing from 0 to 100 nt in steps of 10 nt make sense?

Reply:
Yes, I think that would be nice.